source/integrations/setup-nginx-proxy-with-minio.rst

```diff
@@ -0,0 +1,193 @@
+      - Proxy requests to the root ``https://minio.example.net`` to the MinIO Server listening on ``https://minio.local:9000``.
+
+      - Proxy requests to the subpath ``https://minio.example.net/minio`` to the MinIO Console listening on ``https://minio.local:9001``.
+
+      The following location blocks provide a template for further customization in your unique environment:
+
+      .. code-block:: shell
+         :class: copyable
+
+         upstream minio {
+            least_conn;
+            server minio-01.internal-domain.com;
+            server minio-02.internal-domain.com;
+            server minio-03.internal-domain.com;
+            server minio-04.internal-domain.com;
+         }
+
+         server {
+            listen       80;
+            listen  [::]:80;
+            server_name  minio.example.net;
+
+            # To allow special characters in headers
+            ignore_invalid_headers off;
+            # Allow any size file to be uploaded.
+            # Set to a value such as 1000m; to restrict file size to a specific value
+            client_max_body_size 0;
+            # To disable buffering
+            proxy_buffering off;
+            proxy_request_buffering off;
+
+            location / {
+               proxy_set_header Host $http_host;
+               proxy_set_header X-Real-IP $remote_addr;
+               proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+               proxy_set_header X-Forwarded-Proto $scheme;
+
+               proxy_connect_timeout 300;
+               # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+               proxy_http_version 1.1;
+               proxy_set_header Connection "";
+               chunked_transfer_encoding off;
+
+               proxy_pass https://minio:9000/; # This uses the upstream directive definition to load balance
+            }
+
+            location /minio {
+               proxy_set_header Host $http_host;
+               proxy_set_header X-Real-IP $remote_addr;
+               proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+               proxy_set_header X-Forwarded-Proto $scheme;
+               proxy_set_header X-NginX-Proxy true;
+
+               # This is necessary to pass the correct IP to be hashed
+               real_ip_header X-Real-IP;
+
+               proxy_connect_timeout 300;
+               
+               # To support websockets in MinIO versions released after January 2023
+               proxy_http_version 1.1;
+               proxy_set_header Upgrade $http_upgrade;
+               proxy_set_header Connection "upgrade";
+               
+               chunked_transfer_encoding off;
+
+               proxy_pass https://minio:9001/; # This uses the upstream directive definition to load balance and assumes a static Console port of 9001
+            }
+         }
+
+   .. tab-item:: Subdomain
+
+      Create or configure a unique subdomain for both the MinIO Server S3 API and MinIO Console Web GUI.
```

Comment:
I'd say `for each of the` here.
"for both" risks implying one subdomain is all that is needed.

Maybe
> Create or configure separate, unique subdomains for the MinIO Server S3 API and for the MinIO Console Web GUI.

Separate and unique may be too much, but I think making subdomains plural really emphasizes they need two subdomains. I'll leave that to you to decide.

Reply:
Taking your suggestion!